runtime/tests/interpreter/interpreter_test.go

```diff
@@ -68,7 +69,14 @@ func parseCheckAndInterpretWithOptions(
 		options.HandleCheckerError(err)
 	} else {
```

Comment:
unnecessary nesting. Join into else if.

Reply:
Merged in 00892183404433aeff0678f368bd400556d343bf